docs/methods.rst

```diff
@@ -114,6 +114,58 @@ To continue a run, specify the ``mcwalkerset`` element before your VMC/DMC block
 
 In the project id section, make sure that the series number is different from any existing ones to avoid overwriting them.
 
+.. _batched_drivers:
+
+Batched drivers
+---------------
+
+Under the Exascale Computing Project effort a new set of QMC drivers was developed
+to eliminate the divergence of CPU and GPU code paths at the QMC driver level and make the drivers CPU/GPU agnostic.
+The divergence came from the the fact that the CPU code path favors executing all the compute tasks within a step
+for one walker and then advance walker by walker. Multiple CPU threads process their own assigned walkers in parallel.
+In this way, walkers are not synchronized with each other and maximal throughout can be achieved on CPU.
+The GPU code path favors executing the same compute task over all the walkers together to maximize GPU throughput.
+This GPU code path choice also minimizes the overhead of dispatching computation and host-device data transfer.
+However, there is only one host thread responsible for handling all the interaction between the host and GPUs.
```

Comment:
What does this sentence about "only one host thread" mean?  What consequences are there for the user?

Reply:
I'm not sure, In a proper CUDA/ROCM implementation of almost anything and in the case of the direct inversion, each crowd has a CUDA stream so you have num crowd independent CPU/GPU communications. If the OMPTarget runtime fails to do such a thing i.e. it doesn't have multiple worker threads doing async transfers and kernel launches that tie into to their task and data dependency abstractions, thats their failure.

Reply:
I was describing the legacy implementation. Updated.